extensions/AbuseFilter/AbuseFilter.class.php

```diff
@@ -974,7 +974,6 @@ public static function takeConsequenceAction( $action, $parameters, $title,
 				$block->setBlocker( $filterUser );
 				$block->mReason = wfMsgForContent( 'abusefilter-blockreason', $rule_desc );
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Replace this legacy wfMsg* function call with one of the new i18n functions. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3ALegacyWfMsgUsage)